src/DynamoCoreWpf/Views/Search/LibraryView.xaml.cs

```diff
@@ -170,7 +171,10 @@ private void ExpandCategory(TreeViewItem sender, NodeCategoryViewModel selectedC
             // If categoryToBeExpanded is null, that means not category button, but class button was clicked.
             // During loop we will find out to which category this clicked class belongs.
             if (categoryToBeExpanded != null)
+            {
                 categoryToBeExpanded.IsExpanded = !categoryToBeExpanded.IsExpanded;
+                hasSelectedClassBeenFound = true;
+            }
 
             // Get expanded categories that should be collapsed.
             var categoriesToBeCollapsed = allExpandedCategories.Remove(categoryToBeExpanded);
```

Comment:
ReSharper told me that `categoriesToBeCollapsed` is no longer used, is that intentional?